Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
```

Comment:
### [byebug](http://github.com/deivid-rodriguez/byebug)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 9.0.6 → 10.0.2

[_~~change-log~~, [source-code](https://github.com/deivid-rodriguez/byebug)_]





<details>
<summary>Commits</summary>

A change of **399** commits. See the full changes on [the compare page](https://github.com/deivid-rodriguez/byebug/compare/v9.0.6...v10.0.2).

These are the first 10 commits:
- (15f1e2a) [Update appveyor with trunk, cleanup](https://github.com/deivid-rodriguez/byebug/commit/15f1e2a95d0fb5bc73853c0935ec0b9c9b39b07b)
- (6469d50) [Merge pull request #347 from deivid-rodriguez/enhance/add_tr…](https://github.com/deivid-rodriguez/byebug/commit/6469d5024d95ebffea7a3bf1cd8b7c10df59bba5)
- (816599a) [Merge pull request #346 from MSP-Greg/appveyor_25](https://github.com/deivid-rodriguez/byebug/commit/816599a912e51d869e511a30361b993789c0b889)
- (953daf0) [Bump overcommit from 0.40.0 to 0.41.0 (#348)](https://github.com/deivid-rodriguez/byebug/commit/953daf06b9bc56d1e856a2175caba3d3f96b8664)
- (0e572cc) [Update CHANGELOG.md (#349)](https://github.com/deivid-rodriguez/byebug/commit/0e572cc7931303702bec01355db4619b6282e64e)
- (bdbe849) [Bump rake from 12.0.0 to 12.1.0 (#351)](https://github.com/deivid-rodriguez/byebug/commit/bdbe849ad022dc5a8b2f796a3067de457a3c4aed)
- (b74c673) [Remove PR template (#354)](https://github.com/deivid-rodriguez/byebug/commit/b74c673ebb3fd1312b4e7959588b375d565992c2)
- (e7ed5d0) [Use equal sign for locked gems (#353)](https://github.com/deivid-rodriguez/byebug/commit/e7ed5d0666142be5301b0386c0441b8be17c66a3)
- (eeb69c4) [Bump rubies in CI (#355)](https://github.com/deivid-rodriguez/byebug/commit/eeb69c48d9e5e95efdf8a4cd8523403a0bf5402f)
- (7b1b729) [Bump rubocop from 0.49.1 to 0.50.0 (#352)](https://github.com/deivid-rodriguez/byebug/commit/7b1b729e3f449d299cec384d9ac2514057f17ef9)


</details>